### ISSUE TITLE
Accessibility: Reduce flashing throughout the game

### ISSUE THREAD

**Issue by Miepee** [MetroidAdvRandomizerSystem/mars-fusion-asm#282]
The Web Content Accessibility guidelines recommend all flashing to be below 3 Hz

> Guideline 2.3 Seizures: Do not design content in a way that is known to cause seizures or physical reactions.
> Three Flashes [or Below Threshold]
> Web pages do not contain anything that flashes more than three times in any one second period[, or the flash is below the general flash and red flash thresholds].

Stuff outside brackets is ideal, stuff inside angled brackets would be compromise.
Source: https://www.w3.org/WAI/WCAG22/quickref/?versions=2.0#qr-seizure-does-not-violate

I'm not sure on all things that can cause flashes besides getting damaged or dealing damage. But would probably be a good idea to do something to minimize those flashes to below 3Hz
3Hz is approximately once every 20 frames. Right now getting damaged from an enemy flashes at 29Hz. 

Will need quite a bit of creativity.

**Comment by Miepee** [comment]
Minimap also flashes quite fast

**Comment by cluekitty** [comment]
Other notable strobing:
* Nightmare (preparing for core-x phase)
* Omega (Defeated and screaming)

**Comment by Miepee** [comment]
Aren't those just loops of "enemy is getting damaged"?

**Comment by cluekitty** [comment]
There are some specific cases where they strobe without user input when moving to a location or playing an animation. Nightmare will continuously strobe until it reaches the target in the room where it transforms into the core-x after you defeat phase 2.

Omega will strobe during its death animation as well until it despawns.
